src/app/routes/ssoRoutes.js

```diff
@@ -43,7 +43,7 @@ module.exports = (app) => {
       req.session.userDetails = userDetails;
       logger.info({msg: "userDetails fetched" + userDetails});
       if(!_.isEmpty(userDetails) && (userDetails.phone || userDetails.email)) {
-        redirectUrl = successUrl + getQueryParams({ id: userDetails.userName });
+        redirectUrl = successUrl + encryptUserData({id: userDetails.userName});
```

Comment:
@deveshMantra - we added this encryption logic in three places. I am unable to understand the code (due to lack of proper code comments & structuring). I am hoping all the scenarios are addressed. However, we still need refactoring of SSO module to make it modular and easy to maintain.
@vinukumar-vs